scripts/github_runner/deploy_runners.go

```diff
@@ -134,6 +134,12 @@ func deployRunner(host string, runnerConf RunnerConf, deployerConf *DeployerConf
 	case "cri":
 		setupCmd = fmt.Sprintf("cd vhive && ./scripts/github_runner/setup_bare_metal_runner.sh %s %s %s", deployerConf.GhOrg,
 			deployerConf.GhPat, runnerConf.Sandbox)
+		var redeploySetupCmd string = fmt.Sprintf("echo '10 4 * * * root rm -rf ./runners/ && %s' >> etc/crontab", setupCmd)
+		out, err = client.Exec(redeploySetupCmd)
+		log.Debug(string(out))
```

Comment:
need more elaborate output

Reply:
Add a line of output indicating what command will be executed